amazon-kinesis-client/src/test/java/software/amazon/kinesis/coordinator/SchedulerTest.java

```diff
@@ -259,6 +271,95 @@ public final void testInitializationFailureWithRetriesWithConfiguredMaxInitializ
         verify(shardDetector, times(maxInitializationAttempts)).listShards();
     }
 
+    /**
+     * Tests if two workers can concurrently sync shards.
+     * Runs the first worker and waits until it starts syncing shards. Then starts the second
+     * worker and checks if the second worker can start syncing shards.
+     */
+    @Test(timeout = 30000L)
+    public final void testWorkersCanSyncShardsInParallel() {
```

Comment:
Avoid multithreaded testing in unit tests. Testing a single instance should be fine for unit testing.

Reply:
OK, will remove this unit test.